mockups/pyGtkMock/guiTry.py

```diff
@@ -0,0 +1,68 @@
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk, Gdk
+from defcon import Font
+
+
+class FlowBoxWindow(Gtk.Window):
+
+    def __init__(self):
+        Gtk.Window.__init__(self, title="Character Map Demo")
+        self.set_border_width(10)
+        self.set_default_size(800, 800)
+
+        header = Gtk.HeaderBar(title="Character Map")
+        header.set_subtitle("Sample Character Map app")
+        header.props.show_close_button = True
+
+        self.set_titlebar(header)
+
+        scrolled = Gtk.ScrolledWindow()
+        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
+
+        flowbox = Gtk.FlowBox()
```

Comment:
I'm not sure where it is.   But it has Gtk 3.6 available.  Flowbox is from 3.12 :(

On 22 May 2016 1:53:11 AM AEST, Dave Crossland <notifications@github.com> wrote:
>> +
>> +    def __init__(self):
>> +        Gtk.Window.__init__(self, title="Character Map Demo")
>> +        self.set_border_width(10)
>> +        self.set_default_size(800, 800)
>> +
>> +        header = Gtk.HeaderBar(title="Character Map")
>> +        header.set_subtitle("Sample Character Map app")
>> +        header.props.show_close_button = True
>> +
>> +        self.set_titlebar(header)
>> +
>> +        scrolled = Gtk.ScrolledWindow()
>> +        scrolled.set_policy(Gtk.PolicyType.NEVER,
>Gtk.PolicyType.AUTOMATIC)
>> +
>> +        flowbox = Gtk.FlowBox()
>
>Where are XO requirements documented?
>
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/sugarlabs/edit-fonts-activity/pull/22/files/e9c4ddb38e51587236fbb82b3481b1a91bb18f20#r64136361

-- 
Sent from my phone. Please excuse my brevity.